src/cron/updateSourcePublicThreshold.ts

```diff
@@ -0,0 +1,27 @@
+import { Cron } from './cron';
+import { Source } from '../entity';
+import { updateFlagsStatement } from '../common';
+
+export const updateSourcePublicThreshold: Cron = {
+  name: 'update-source-public-threshold',
+  handler: async (con, logger) => {
+    logger.info('updating public threshold value for valid sources...');
+    const result = await con
+      .getRepository(Source)
+      .createQueryBuilder('s')
+      .update()
+      .set({ flags: updateFlagsStatement({ publicThreshold: true }) })
+      .where(
+        `
+          (flags->>'totalMembers')::int >= 3 AND
+          (flags->>'totalPosts')::int >= 3 AND
```

Comment:
I'm missing squad image and description
https://dailydotdev.slack.com/archives/C069XNYMDGV/p1726986254903039?thread_ts=1726896915.493839&cid=C069XNYMDGV

Reply:
Should be part of the conditions now.